plugins/flowdock/lib/samson_flowdock/samson_plugin.rb

```diff
@@ -0,0 +1,43 @@
+module SamsonFlowDock
+  class Engine < Rails::Engine
+  end
+end
+
+Samson::Hooks.callback :stage_defined do
+  Stage.class_eval do
+    has_many :flowdock_flows
```

Comment:
added an example migration

On Tue, Mar 3, 2015 at 6:31 PM, Roman Sandler <notifications@github.com>
wrote:

> In plugins/flowdock/lib/samson_flowdock/samson_plugin.rb
> <https://github.com/zendesk/samson/pull/259#discussion_r25748136>:
>
> > @@ -0,0 +1,43 @@
> > +module SamsonFlowDock
> > +  class Engine < Rails::Engine
> > +  end
> > +end
> > +
> > +Samson::Hooks.callback :stage_defined do
> > +  Stage.class_eval do
> > +    has_many :flowdock_flows
>
> since this will be the canonical example, can we work out where the
> migration will live and how it will run?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/259/files#r25748136>.
>